src/Examine.Test/Examine.Lucene/Search/FluentApiTests.cs

```diff
@@ -1427,6 +1427,38 @@ public void Skip_Results_Returns_Different_Results()
             }
         }
 
+        [Test]
+        public void TimeLimit_Applies()
+        {
+            var analyzer = new StandardAnalyzer(LuceneInfo.CurrentVersion);
+            using (var luceneDir = new RandomIdRAMDirectory())
+            using (var indexer = GetTestIndex(luceneDir, analyzer))
+            {
+                indexer.IndexItems(new[] {
+                    ValueSet.FromObject(1.ToString(), "content",
+                        new { nodeName = "umbraco", headerText = "world", writerName = "administrator" }),
+                    ValueSet.FromObject(2.ToString(), "content",
+                        new { nodeName = "umbraco", headerText = "umbraco", writerName = "administrator" }),
+                    ValueSet.FromObject(3.ToString(), "content",
+                        new { nodeName = "umbraco", headerText = "umbraco", writerName = "administrator" }),
+                    ValueSet.FromObject(4.ToString(), "content",
+                        new { nodeName = "hello", headerText = "world", writerName = "blah" })
+                    });
+
+                var searcher = indexer.Searcher;
+
+                //Arrange
+                var sc = searcher.CreateQuery("content").Field("writerName", "administrator");
+
+                //Act
+                var results = sc.Execute(new QueryOptions(0, 5, TimeSpan.FromMilliseconds(100)));
+
+                //Assert
+                Assert.AreNotEqual(results.First(), results.Skip(2).First(), "Third result should be different");
```

Comment:
What is this asserting with regards to the time limit?

Reply:
This only checks that the query will execute with a time limit set.